sdk/security_keyvault/Cargo.toml

```diff
@@ -14,15 +14,14 @@ edition = "2021"
 
 [dependencies]
 base64 = "0.13"
-thiserror = "1.0"
 reqwest = { version = "0.11", features = ["json"] }
 chrono = { version = "0.4", features = ["serde"] }
 const_format = "0.2.13"
 serde_json = "1.0"
 url = "2.2"
 serde = { version = "1.0", features = ["derive"] }
 getset = "0.1"
-azure_core = { path = "../core", version = "0.2" }
+azure_core = { path = "../core", version = "0.2.1" }
```

Comment:
Why do you pick a specific version here?

Reply:
I'll revert that back.  I changed it to match the version number I had locally, in a vain attempt to fix some build issues (which turned out to be because I hadn't properly synced the latest code).  Functionally there won't be any difference (as 0.2.0 was yanked, and specifying 0.2.1 will match the latest 0.2.x available).  However I think it is clearer to not specify the patch version.